src/style.css

```diff
@@ -0,0 +1,41 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  width: 96px;
+  height: 96px;
+
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+
+  height: 16px;
+  width: 16px;
+  background: url(../src/images/star.svg) no-repeat;
+}
+
+.stars--5 > :nth-last-child(n+1) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--4 > :nth-last-child(n+2) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--3 > :nth-last-child(n+3) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--2 > :nth-last-child(n+4) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--1 > :nth-last-child(n+5) {
+  background-image: url(./images/star-active.svg);
+}
```

Comment:
Check point 2 in [checklist](https://github.com/mate-academy/layout_stars/blob/master/checklist.md)